radio/src/dataconstants.h

```diff
@@ -284,8 +284,8 @@ enum BeeperMode {
     TRAINER_MODE_SLAVE,
 #if defined(PCBTARANIS)
     TRAINER_MODE_MASTER_SBUS_EXTERNAL_MODULE,
-    TRAINER_MODE_MASTER_CPPM_EXTERNAL_MODULE,
 #endif
+    TRAINER_MODE_MASTER_CPPM_EXTERNAL_MODULE,
```

Comment:
this needs an EEPROM conversion!

Reply:
@bsongis Is it still possible to add to 2.3? What exactly is necessary?